### ISSUE TITLE
Runestone Issue: accessibility issues in Horizontal Parsons

### ISSUE THREAD

**Issue by ascholerChemeketa** [RunestoneInteractive/rs#1197]
Horizontal Parsons problems lack essential accessibility features.

Both block areas should be tab stops. Within an area, arrows should move from block to block and space/enter should move the block to the other area.

There also should be non-visual feedback while navigating between blocks and when blocks are moved.
